server/config/config.go

```diff
@@ -777,6 +825,12 @@ func (man Manager) addConfigs() {
 	man.addConfigString("mdm_apple.mdm.push.key_pem", "", "MDM APNS PEM-encoded private key")
 	man.addConfigString("mdm_apple.dep.token", "", "MDM DEP Auth Token")
 	man.addConfigDuration("mdm_apple.dep.sync_periodicity", 1*time.Minute, "How much time to wait for DEP profile assignment")
+
+	// MDM config
+	man.addConfigString("mdm.apple_apns_cert", "", "Apple APNs PEM-encoded certificate path")
+	man.addConfigString("mdm.apple_apns_key", "", "Apple APNs PEM-encoded private key path")
+	man.addConfigString("mdm.apple_scep_cert", "", "Apple SCEP PEM-encoded certificate path")
+	man.addConfigString("mdm.apple_scep_key", "", "Apple SCEP PEM-encoded private key path")
```

Comment:
probably also a question for @zhumo , should we hide these config options from the output of `--help` until we officially release MDM?

we did something similar for the sandbox flags:

https://github.com/fleetdm/fleet/blob/5fb0189a8fc7cb128a9a483e817001f536c52c13/server/config/config.go#L602-L606

Reply:
Yeah let's do the same...

Reply:
Updated with the mdm flags hidden.